test.js

```diff
@@ -18,14 +18,15 @@ test('Should return 503 on maxEventLoopDelay', t => {
     reply.send({ hello: 'world' })
   })
 
-  fastify.listen(0, err => {
+  fastify.listen(0, (err, address) => {
     t.error(err)
+    fastify.server.unref()
 
     // Increased to prevent Travis to fail
     process.nextTick(() => sleep(1000))
     sget({
       method: 'GET',
-      url: 'http://localhost:' + fastify.server.address().port
+      url: `${address}`
```

Comment:
You can use just `address`, there is no need to use a template string :)

Reply:
@delvedor removed template strings where not really needed ... thanks for the feedback